templates/proxy-deployment.yaml

```diff
@@ -71,11 +71,9 @@ spec:
         - name: AUTHENTICATE_SERVICE_URL
           value: {{ default (printf "https://authenticate.%s" .Values.config.rootDomain ) .Values.proxy.authenticateServiceUrl }}
         - name: AUTHORIZE_SERVICE_URL
-          value: {{ default (printf "https://authorize.%s" .Values.config.rootDomain ) .Values.proxy.authorizeServiceUrl }}
+          value: {{ default (printf "https://%s.%s.svc.cluster.local" (include "pomerium.authorize.fullname" .) .Release.Namespace ) .Values.proxy.authorizeInternalUrl}}
```

Comment:
See https://github.com/pomerium/pomerium/issues/153 by the way

Reply:
Should be fixed by https://github.com/pomerium/pomerium/pull/154 thx!